tests/conftest.py

```diff
@@ -148,3 +148,13 @@ def plt():
     from matplotlib import pyplot as plt
 
     return plt
+
+
+@pytest.fixture
+def adata_for_test():
+    """Fixture to provide a preprocessed AnnData object for testing."""
+    import scanpy as sc
+
+    adata = sc.datasets.pbmc68k_reduced()
+    sc.pp.neighbors(adata)
+    return adata
```

Comment:
Please move this into `tests/test_cluster_resolution.py`, and use the cached version:

```
from testing.scanpy._helpers.data import pbmc68k_reduced
```

instead of `sc.datasets.pbmc68k_reduced`.